tests/test_functons.py

```diff
@@ -0,0 +1,47 @@
+from codecademy.functions import one_good_turn
+from codecademy.functions import square
+from codecademy.functions import deserves_another
+from codecademy.functions import biggest_number
+
+import pytest, sys
```

Comment:
just for your information per PEP8 python style guide each import has to be on a separate line.
Also an empty line should be between import from a standard library (sys) and 3rd party lib (pytest) and then local imports
https://www.python.org/dev/peps/pep-0008/#imports